api/index.js

```diff
@@ -13,15 +13,22 @@ app.use(express.json());
 app.use(free_the_cors());
 app.use(braidify);    // Add braid stuff to req and res
 
-const board = {};
-for (let i = 1; i < 11; i++) {
-  for (let j = 1; j < 11; j++) {
-    board[`${i}_${j}`] = ".";
+const boards = {}
+
+const generateBoard = (url) => {
+  boards[url] = {}
+  for (let i = 1; i < 11; i++) {
+    for (let j = 1; j < 11; j++) {
+      boards[url][`${i}_${j}`] = ".";
+    }
   }
 }
 
-app.get("/api/board", (req, res) => {
+
+app.get("/api/:url", (req, res) => {
```

Comment:
in the GET, POST, and PUT request, change the `url` variable to `topic`. We're already using a variable called `url` in our for loops that `sendVersion` to be subscriber so using another variable called `url` could lead to confusion. plus `topic` is a more suitable name, since we're building a pub-sub like system, where `topic` is typically the term used.